big_tests/tests/metrics_session_SUITE.erl

```diff
@@ -113,16 +111,23 @@ auth_failed(Config) ->
 session_global(Config) ->
     escalus:story(Config, [{alice, 1}], fun(_Alice) ->
 
-        timer:sleep(?GLOBAL_WAIT_TIME),
-        assert_counter(global, 1, totalSessionCount)
+        wait_for_counter(global, 1, totalSessionCount)
 
         end).
 
 session_unique(Config) ->
     escalus:story(Config, [{alice, 2}], fun(_Alice1, _Alice2) ->
 
-        timer:sleep(?GLOBAL_WAIT_TIME),
-        assert_counter(global, 1, uniqueSessionCount),
-        assert_counter(global, 2, totalSessionCount)
+        wait_for_counter(global, 1, uniqueSessionCount),
+        wait_for_counter(global, 2, totalSessionCount)
 
         end).
+
+
+wait_for_counter(ExpectedValue, Counter) ->
```

Comment:
As far as I can tell, this is the exact copy of the function you defined [here](https://github.com/esl/MongooseIM/pull/1982/files?utf8=%E2%9C%93&diff=unified#diff-e51d3c9fb9c07615eb6d1ded14daa8e5R270)

Reply:
Indeed, it is.
@zofpolkowska Please move this function to `metrics_helper` and call appropriately from both this suite and `metrics_c2s_SUITE`.